pkgs/development/python-modules/pysaml2/default.nix

```diff
@@ -21,7 +22,16 @@
 , xmlsec
 }:
 
-buildPythonPackage rec {
+let
+  pymongo3 = pymongo.overridePythonAttrs(old: rec {
+    version = "3.12.3";
+    src = fetchPypi {
+      pname = "pymongo";
+      inherit version;
+      sha256 = "sha256-ConK3ABipeU2ZN3gQ/bAlxcrjBxfAJRJAJUoL/mZWl8=";
+    };
+  });
```

Comment:
This is not allowed in pythonPackages if the import name is the same as pymongo because if both packages end up in the same python env things will break.